package.json

```diff
@@ -58,6 +58,15 @@
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.8",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.54.1",
+    "eslint": "^8.0.1",
```

Comment:
Are these dependencies needed if we're not running ESLint through yarn?

Reply:
I don't think so. These dev dependencies should be needed if we linting the code locally.

Reply:
I think these should be taken out. They are not needed if you lint locally with the CodeClimate CLI.